project-icp/bee-pollinator-app
# Implement forage and sort dropdown selectors on sidebar

## Overview

Continuing to build out the frontend, I converted the HTML dropdowns into react component dropdowns. These dropdowns will change the data and data arrangement in the apiary cards, eventually. Both dropdowns kick off their associated action (see gif) but you will only see data update in the sidebar when toggling the forage range dropdown.

Connects #316 

### Demo

![dropdownselectsbees](https://user-images.githubusercontent.com/10568752/48139228-f7203680-e273-11e8-8119-2393b69e8ce0.gif)

### Notes

- The styling follows the prototype, which means there's a sllight discrepency with the wireframe
-  I changed the eslint config because the eslint rule the form format was complaining about was satisfied, there are several related issues out there see: https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/302
- We don't know what the sorts will be, so for now that dropdown is pretty sparse
- I've updated the s3 bucket and file naming from say '3k' --> '3km' 

## Testing Instructions

Pull down the branch and toggle the dropdowns. See the actions in the console and the data change for forage range toggling.
